packages/rug/src/utils.js

```diff
@@ -26,10 +34,35 @@ export const calcModularScale = curry((scale, exponent) =>
   Math.round(base * Math.pow(scale, Math.round(exponent))) / base
 )
 
+/**
+ * Color Utilities
+ */
+
 export const addUnit = curry((unit, value) => `${value}${unit}`)
 
-export const opacify = (o, rgb) => rgb.replace('b(', 'ba(').replace(')', `,${o})`)
+export const opacify = (rgb, o = 1) => rgb.replace('b(', 'ba(').replace(')', `,${o})`)
```

Comment:
The signature was initially opacity first and curried to allow the creation of opacity functions.

e.g.
```js
const opacity = {
  light: opacify(0.4)
  lighter: opacify(0.4)
}

style = {
  color: opacity.light(color.charcoal)
}
```

I did like this pattern. Fine if we don't want to go there, but just so you are aware if making this change.